src/ui/properties/FloorPlanNodeEditor.js

```diff
@@ -72,82 +88,93 @@ class FloorPlanNodeEditor extends Component {
-            smallStep={0.001}
-            mediumStep={0.01}
-            largeStep={0.1}
-            min={0.1}
-            displayPrecision={0.0001}
-            onChange={this.onChangeCellSize}
-          />
+        {node.navMeshMode === NavMeshMode.Automatic ? (
+          <>
+            <InputGroup name="Auto Cell Size">
+              <BooleanInput value={node.autoCellSize} onChange={this.onChangeAutoCellSize} />
+            </InputGroup>
+            {!node.autoCellSize && (
+              <NumericInputGroup
+                name="Cell Size"
+                value={node.cellSize}
+                smallStep={0.001}
+                mediumStep={0.01}
+                largeStep={0.1}
+                min={0.1}
+                displayPrecision={0.0001}
+                onChange={this.onChangeCellSize}
+              />
+            )}
+            <NumericInputGroup
+              name="Cell Height"
+              value={node.cellHeight}
+              smallStep={0.001}
+              mediumStep={0.01}
+              largeStep={0.1}
+              min={0.1}
+              onChange={this.onChangeCellHeight}
+              unit="m"
+            />
+            <NumericInputGroup
+              name="Agent Height"
+              value={node.agentHeight}
+              smallStep={0.001}
+              mediumStep={0.01}
+              largeStep={0.1}
+              min={0.1}
+              onChange={this.onChangeAgentHeight}
+              unit="m"
+            />
+            <NumericInputGroup
+              name="Agent Radius"
+              value={node.agentRadius}
+              min={0}
+              smallStep={0.001}
+              mediumStep={0.01}
+              largeStep={0.1}
+              onChange={this.onChangeAgentRadius}
+              unit="m"
+            />
+            <NumericInputGroup
+              name="Maximum Step Height"
+              value={node.agentMaxClimb}
+              min={0}
+              smallStep={0.001}
+              mediumStep={0.01}
+              largeStep={0.1}
+              onChange={this.onChangeAgentMaxClimb}
+              unit="m"
+            />
+            <NumericInputGroup
+              name="Maximum Slope"
+              value={node.agentMaxSlope}
+              min={0.00001}
+              max={90}
+              smallStep={1}
+              mediumStep={5}
+              largeStep={15}
+              onChange={this.onChangeAgentMaxSlope}
+              unit="°"
+            />
+            <NumericInputGroup
+              name="Minimum Region Area"
+              value={node.regionMinSize}
+              min={0.1}
+              smallStep={0.1}
+              mediumStep={1}
+              largeStep={10}
+              onChange={this.onChangeRegionMinSize}
+              unit="m²"
+            />
+          </>
+        ) : (
+          <InputGroup name="Custom Navmesh Url">
+            <ModelInput value={node.navMeshSrc} onChange={this.onChangeNavMeshSrc} />
```

Comment:
Not new to this PR, just anything taking a model in general so not something we need to address in this PR, but we might want to think about how we could visually indicate somehow that this field can have models dragged into it from assets. Ideally there might even be a little button on the field that pops up an asset picker (similar to how it works in Unity)